ci/azure-pipelines.yml

```diff
@@ -22,17 +43,29 @@ jobs:
         sudo apt-get install build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libreadline-dev libffi-dev wget libsqlite3-dev
       displayName: Install Python components
 
-    - task: UsePythonVersion@0
-      inputs:
-        versionSpec: '3.7' 
-        addToPath: true
-        
     - script: |
-        python3 --version
-        pip3 install --upgrade pip
+        pip install wheel
+        pip3 install wheel
         pip3 install --upgrade setuptools
+        pip3 uninstall crypto
+        pip3 uninstall pycrypto
+        pip install pycrypto
+        pip3 install pycrypto
```

Comment:
Uninstall then reinstall pycrypto? Why?

Reply:
because it is 'installed' already, and I double check with apt-get, but whey a python script tries to include it, it isn't found:
```python
    from Crypto.PublicKey import RSA
ModuleNotFoundError: No module named 'Crypto'
```
This was the nicest way I could figure out how to deal with it. There is also apparently a possible name collision between crypto and pycrypto which this sure clear up.

This only occurs when using python 3.6 or 3.7, leaving the default python 3.3 in place seems to work fine but is incompatible with MU.